llvm/tools/sycl-post-link/CompileTimePropertiesPass.h

```diff
@@ -14,9 +14,11 @@
 
 #pragma once
 
+#include "llvm/IR/IntrinsicInst.h"
```

Comment:
There are only uses of `IntrinsicInst *` in the header. Will the forward declaration of the `IntrinsicsInst` be enough?